python/tvm/relay/frontend/tensorflow.py

```diff
@@ -77,10 +77,21 @@ def _dim_check(attrs):
         return False
     return _dim_check, "Only 2d or 3d kernel supported."
 
+# Build count_used to record params have been converted to attrs
+used_params = {}
```

Comment:
Only need set?

Reply:
Agree with @srkreddy1238 if relay build can remove unused params, we don't  need this here.

Reply:
Thank you for the suggestions! Then I think the thing to do is simple and we can just remove  the pop operations.

Reply:
Hi, I have already modified the code according to your suggestions! Thank you for the help!